core-nonfree/story-api-nonfree/src/main/java/edu/cmu/cs/dennisc/nebulous/GenericModelAdapter.java

```diff
@@ -86,15 +87,14 @@ protected void renderGeometry(RenderContext rc, GlrVisual.RenderType renderType)
   }
 
   @Override
```

Comment:
I just want to say, I find this new code more self-documenting than before ;)

Reply:
I can add helpful comments like:
`// Trying to do the same thing it was doing, but in a little better way.`